docs/pages/packages/format.mdx

```diff
@@ -8,22 +8,10 @@ It follows the [formatter spec](https://console.spec.whatwg.org/#formatter) and
 
 ## Installation
 
-<Tabs items={['npm', 'yarn', 'pnpm']} storageKey="selected-pkg-manager">
-  <Tab>
-    ```sh
-    npm install @edge-runtime/format --save
-    ```
-  </Tab>
-  <Tab>
-    ```sh
-    yarn add @edge-runtime/format
-    ```
-  </Tab>
-  <Tab>
-    ```sh
-    pnpm install @edge-runtime/format --save
-    ```
-  </Tab>
+<Tabs items={['npm', 'yarn', 'pnpm']} storageKey='selected-pkg-manager'>
+  <Tab>```sh npm install @edge-runtime/format --save ```</Tab>
```

Comment:
This code block will look like following (`sh` is not recognized as language name!):

```sh npm install @edge-runtime/format --save ```

IIRC formatter does this broken formatting.

Reply:
ah right will revert all changes to `docs/pages/**/*.mdx` <picture data-single-emoji=":relieved-telegram:" title=":relieved-telegram:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/relieved-telegram/c1d17d63d72a070c.gif" alt=":relieved-telegram:" align="absmiddle"></picture>

Reply:
<picture data-single-emoji=":ditto:" title=":ditto:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/ditto/f00855b29aa73a51.png" alt=":ditto:" align="absmiddle"></picture>